wdl/GatherSampleEvidence.wdl

```diff
@@ -23,6 +23,7 @@ workflow GatherSampleEvidence {
     # Evidence collection flags
     Boolean collect_coverage = true
     Boolean collect_pesr = true
+    Boolean move_bam_or_cram_files = false
```

Comment:
Can you add a quick comment here explaining when this should be used? tldr; Google Cloud Platform users can safely enable and get a slight cost improvement. Users running on Azure or shared filesystems should NOT enable.

Reply:
That will be helpful! I added the comment with one modification: both GCP & Azure users can safely enable this, and only users running on shared file systems are _not_ recommended to enable it.